manifests/kustomize/env/namespaced-installation/pipeline-application.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: app.k8s.io/v1beta1
```

Comment:
we can change the base application to a base and a json patch for adding resources, so that we only need to include the base here

Reply:
I didn't notice this file was checked in. I think I can remove application.yaml in this manifest